apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -1,238 +1,135 @@
+import { Button } from '_app/shared/ButtonUI';
+import BottomMenuLayout, {
+    Content,
+    Menu,
+} from '_app/shared/bottom-menu-layout';
+import ActiveCoinsCard from '_components/active-coins-card';
+import { SuiIcons } from '_components/icon';
+import Overlay from '_components/overlay';
 import { parseAmount } from '_helpers';
-import {
-    useAppSelector,
-    useAppDispatch,
-    useCoinDecimals,
-    useIndividualCoinMaxBalance,
-} from '_hooks';
-import {
-    accountAggregateBalancesSelector,
-    accountCoinsSelector,
-} from '_redux/slices/account';
+import { useAppSelector, useCoinDecimals } from '_hooks';
+import { accountCoinsSelector } from '_redux/slices/account';
 import { Coin } from '_redux/slices/sui-objects/Coin';
-import { sendTokens } from '_redux/slices/transactions';
-import { trackEvent } from '_src/shared/plausible';
 import { useGasBudgetInMist } from '_src/ui/app/hooks/useGasBudgetInMist';
-import PageTitle from '_src/ui/app/shared/PageTitle';
 
-import type { SerializedError } from '@reduxjs/toolkit';
-import type { FormikHelpers } from 'formik';
-
-import st from './TransferCoinPage.module.scss';
-
-const initialValues = {
-    to: '',
-    amount: '',
-};
-
-export type FormValues = typeof initialValues;
+import type { SubmitProps } from './SendTokenForm';
 
 const DEFAULT_FORM_STEP = 1;
 
-// TODO: show out of sync when sui objects locally might be outdated
 function TransferCoinPage() {
     const [searchParams] = useSearchParams();
+    const [showModal, setShowModal] = useState(true);
+
     const coinType = searchParams.get('type');
-    const aggregateBalances = useAppSelector(accountAggregateBalancesSelector);
-    const coinBalance = useMemo(
-        () => (coinType && aggregateBalances[coinType]) || BigInt(0),
-        [coinType, aggregateBalances]
-    );
-    const gasAggregateBalance = useMemo(
-        () => aggregateBalances[SUI_TYPE_ARG] || BigInt(0),
-        [aggregateBalances]
-    );
-    const coinSymbol = useMemo(
-        () => (coinType && CoinAPI.getCoinSymbol(coinType)) || '',
-        [coinType]
-    );
+    const [currentStep, setCurrentStep] = useState<number>(DEFAULT_FORM_STEP);
+    const [formData, setFormData] = useState<SubmitProps>();
     const allCoins = useAppSelector(accountCoinsSelector);
     const allCoinsOfTransferType = useMemo(
         () => allCoins.filter((aCoin) => aCoin.type === coinType),
         [allCoins, coinType]
     );
-    const [sendError, setSendError] = useState<string | null>(null);
-    const [currentStep, setCurrentStep] = useState<number>(DEFAULT_FORM_STEP);
-    const [formData] = useState<FormValues>(initialValues);
     const [coinDecimals] = useCoinDecimals(coinType);
-    const [gasDecimals] = useCoinDecimals(SUI_TYPE_ARG);
-    const [amountToSend, setAmountToSend] = useState(BigInt(0));
-    const maxSuiSingleCoinBalance = useIndividualCoinMaxBalance(SUI_TYPE_ARG);
+
     const gasBudgetEstimationUnits = useMemo(
```

Comment:
I think we can move this inside the SendTokenForm and expose it with the other values when we submit the form

Reply:
Please see the other PR